CXX_for_OpenCL.txt

```diff
@@ -8,7 +8,7 @@ Khronos{R} OpenCL Working Group
 :data-uri:
 :icons: font
 :toc2:
-:toclevels: 2
+:toclevels: 4
```

Comment:
Any reason to have 4?  The OpenCL C spec uses 3.

Reply:
No big reason. Now updated to 3. :)